### DISCUSSION TITLE
VNC4windows?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5618] [Q&A] [answered]
Hi, for some reasons we need for some PC VNC. 
Is it possible to use it therefor? Is there a doc? I din't find a video nor a pdf/html file.

**Comment by si458** [comment, marked as answer]
1. make sure the remote device has the meshagent installed on it
(only needed if you want it, but can also work with local groups, and relay groups, just make sure 5900 is open on the remote firewall)
2. install VNC software of some kind on the remote pc (realvnc, tigervnc,tightvnc, OSX built in, LINUX BUILT IN, etc...)
3. make sure its using port 5900
4. go to the device in the web ui
5. click 'Web-VNC', it will open the novnc web ui, click connect, enter user/pass, away you go!
